recipes/oddt/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "oddt" %}
+{% set version = "0.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 330f510a6563c371627370200fbfb39ed3496662d067d17e09edb5a6025c8ae6
+
+build:
+  skip: true  # [win]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - six
```

Comment:
Are all these dependences needed in `host`? Usually only `python` and `pip` are enough.

Reply:
The package is imported during the setup it looks like they are needed: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=131372&view=logs&j=10bda42a-d420-5c28-a43f-bc4e66405e5c&t=bef775d0-a1a2-55a3-9721-17147029baf0&l=479

Reply:
I put the deps back. Let me know if I should do something else.

Reply:
Thanks for checking that out @hadim.